app/helpers/alchemy/elements_helper.rb

```diff
@@ -78,7 +78,7 @@ def render_elements(options = {}, &blk)
 
       finder = options[:finder] || Alchemy::ElementsFinder.new(options)
 
-      page_version = if @preview_mode
+      page_version = if Current.preview_page?
```

Comment:
I really like this, and I was wondering whether we should have `@preview_mode` also set to `Current.preview_page?`. Currently, it's just set to `true` in the admin pages controller, which just works (tm).

Reply:
My plan was to not change any legacy code. I am imagining the remove the instance var some day.